package.json

```diff
@@ -103,6 +103,11 @@
       "pdf": {
         "description": "Set to `true` to get index.pdf generated next to your index.html",
         "default": false
+      },
+      "config": {
+        "description": "Stringified JSON or path to JSON file to override [React component config](https://github.com/asyncapi/asyncapi-react/blob/next/docs/configuration/config-modification.md#definition)",
```

Comment:
```suggestion
        "description": "Stringified JSON or a path to a JSON file to override default React component config. The default config and passed via patameter is merged using the [JSON Merge Patch](https://tools.ietf.org/html/rfc7386) algorithm.",
```

Reply:
I didn't use the exact same text as "The default config and passed via patameter is merged using..." but let me know if what I came up with is acceptable.

Reply:
My English is not perfect, so thanks a lot! :)